source/_posts/2015-03-26-creating-a-dynamic-single-page-app-for-our-genome-team-using-react.markdown

```diff
@@ -0,0 +1,96 @@
+---
+layout: post
+title: "Creating a dynamic single page app for our Genome Team using React"
+date: 2015-03-26 10:42
+comments: true
+categories: [React]
+author: Sarah Weir
+github-url: https://www.github.com/sweir27
+twitter-url: http://twitter.com/sweir27
+---
+We recently picked up a Rails application that was a few bugs away from completion. This application allows our [Genome Team](https://www.artsy.net/about/the-art-genome-project) to genome (classify artworks based on visual and art historical characteristics) multiple artworks at once. These characteristics, or "genes" can be added, removed and changed for any of the artworks on the panel. There are many ways to find and add genes to an artwork in this app. For example, a genomer may pull up an existing artist genome to use as a 'template' from which they can batch apply certain genes to multiple artworks. With these and other dynamic views required, our consumers desired an interaction-heavy single page app.
```

Comment:
> This application allows our [Genome Team](https://www.artsy.net/about/the-art-genome-project) to genome (classify artworks based on visual and art historical characteristics) multiple artworks at once

This application allows our [Genome Team](https://www.artsy.net/about/the-art-genome-project)  to classify multiple artworks based on visual and art historical characteristics.